meta/main.yml

```diff
@@ -28,12 +28,13 @@ galaxy_info:
   galaxy_tags:
     - monitoring
     - logging
-    - ops-agent
+    - ops
+    - agent
     - google
-    - google-cloud
+    - cloud
     - collectd
     - fluentd
-    - fluent-bit
-    - open-telemetry
-    - agent
+    - fluentbit
+    - open
```

Comment:
OpenTelemetry seems to be one word in their official site. Maybe opentelemetry as the tag then?

Reply:
Done.